x-pack/plugins/session_view/common/constants.ts

```diff
@@ -13,7 +13,7 @@ export const PROCESS_EVENTS_INDEX = 'logs-endpoint.events.process-default';
 export const PREVIEW_ALERTS_INDEX = '.preview.alerts-security.alerts-default';
 export const ENTRY_SESSION_ENTITY_ID_PROPERTY = 'process.entry_leader.entity_id';
 export const ALERT_UUID_PROPERTY = 'kibana.alert.uuid';
-export const KIBANA_DATE_FORMAT = 'MMM DD, YYYY @ hh:mm:ss.SSS';
+export const KIBANA_DATE_FORMAT = 'MMM DD, YYYY @ hh:mm:ss.SSSSSS';
```

Comment:
are we going to show timestamps in nanoseconds?

Reply:
ha, no, i'll revert this, was just a sanity check when i was fighting some of this jump to cursor stuff.